### DISCUSSION TITLE
Get Pages Count

### DISCUSSION THREAD

**Discussion by siro-invofox** [Stirling-Tools/Stirling-PDF#1796] [Q&A] [answered]
Hi,

I want to know if there's currently a way to get the number of pages of a given PDF.
I've checked the doc and found the method /filter/filter-page-count which allows to check if the pageCount is higher or lower than a given number, but I'm unable to find a method to retrieve directly the pageCount of the file.

Thank you in advance!

**Comment by Frooodle** [comment, marked as answer]
Not currently
Currently we don't have any APIs of that natureb
